src/compile-cache.js

```diff
@@ -214,18 +217,21 @@ exports.install = function (resourcesPath, nodeRequire) {
     return this.rawStack
   }
 
-  Object.keys(COMPILERS).forEach(function (extension) {
-    var compiler = COMPILERS[extension]
-
-    Object.defineProperty(nodeRequire.extensions, extension, {
-      enumerable: true,
-      writable: false,
-      value: function (module, filePath) {
-        var code = compileFileAtPath(compiler, filePath, extension)
-        return module._compile(code, filePath)
-      }
+  // require.extensions will not be set if this is running in a browser.
+  if (require.extensions != null) {
```

Comment:
Hmm, this should probably be `nodeRequire.extensions` instead. I'm still waiting for Atom to build so I can test this.